tests/cpp/test_pointwise.cpp

```diff
@@ -1367,4 +1367,63 @@ TEST_F(NVFuserTest, DomainMapBroadcastIssue3653) {
       executor_cache.fusion(), out_tensors, {t0, t1}, __LINE__, __FILE__);
 }
 
+TEST_F(PointwiseTest, InnerDimAllocationTransformationOnProducer) {
+  for (bool inner_split : {true, false}) {
+    auto fusion = std::make_unique<Fusion>();
+    FusionGuard fg(fusion.get());
+    TensorView* in = makeContigTensor(2);
+    in->split(1, 4, inner_split); // outer split
+    in->setAllocationDomain(in->getLoopDomain(), true);
+    TensorView* out = cos(in);
+    fusion->addInput(in);
+    fusion->addOutput(out);
+    EXPECT_EQ(
+        scheduler_utils::getInputsOutputsWithInnerDim(out, true, false).size(),
```

Comment:
Can you code comment what variables `true` and `false` correspond for easier readability? Same for the below tests.